content/news/2025-01-22_rocur-relaunch/index.en.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "We're Launching the R-Ladies Rotating Curation (RoCur) Account on Bluesky!"
+author: "RoCur team"
+type: blog
+date: "2025-01-22"
```

Comment:
needs to be changed.

Reply:
Author changed; does anything else need to be changed here?

Reply:
i updated the publishing date to 1/24 (i pushed the commit)